configure.sh

```diff
@@ -141,20 +141,26 @@ function configure() {
     CONTAINER_MOUNT_OPTS=:Z
   fi
 
-  if [[ -n "$arg_container_engine" ]]; then
```

Comment:
this can be made without introducing another level of nesting, e.g.:
```bash
if [[ -z "$steamrt_image" ]]; then
    stat "Using bare-metal build."
   # whatever special handling is necessary
elif [[ -n "$arg_container_engine" ]]; then
```